### PR TITLE
Adapt Covid info depending on countries + i18n

### PR DESCRIPTION
**Pull request by G-Ray**
- Ability to set a specific warning message for each country code.
- France has a specific warning, meanwhile a more generic message is displayed for other countries.
- i18n for Covid19.

### CODE REVIEW

File: src/panel/poi/PoiBlockContainer.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default class PoiBlockContainer extends React.Component {
     const displayCovidInfo = this.props.covid19Enabled && blocks.find(b => b.type === 'covid19');
 
     return <div className="poi_panel__info">
-      {displayCovidInfo && <CovidBlock block={covidBlock} />}
+      {displayCovidInfo && <CovidBlock block={covidBlock} poi={this.props.poi} />}
```
Comment (bbecquet) [inline src/panel/poi/PoiBlockContainer.jsx:30]: Maybe we can pass only the country code as props, so the `CovidBlock` doesn't need to know how to access this info from a full POI object.

File: src/panel/poi/blocks/Covid19.jsx [hunk 2]
```diff
@@ -66,36 +70,61 @@ const getContent = ({ status, opening_hours, note, contribute_url }) => {
...
         <p>
-          Pendant toute la période de confinement, se déplacer vers ce lieu n'est autorisé
-          qu'en possession d'une attestation de déplacement dérogatoire.
+          {_('During the entire period of containment, movement to this location is only permitted with an overriding movement certificate.', 'covid19')}
         </p>
         <p>
-          Plus d'informations sur{' '}
-          <a rel="noopener noreferrer" href={covidConf.frInformationUrl}>interieur.gouv.fr</a>
+          {_('More information at', 'covid19')}
+          <a rel="noopener noreferrer" href={covidConf.frInformationUrl}> interieur.gouv.fr</a>
```
Comment (bbecquet) [inline src/panel/poi/blocks/Covid19.jsx:83]: The space shouldn't be part of the link. If you need one, you can use `{' '}` or put the link on the same line.

File: src/panel/poi/blocks/Covid19.jsx [hunk 3]
```diff
@@ -66,36 +70,61 @@ const getContent = ({ status, opening_hours, note, contribute_url }) => {
...
+    <i className="icon-alert-triangle" />
+    <LocalizedWarning countryCode={countryCode} />
+  </div>
+)
+
+const Status = ({ status }) => {
+  const statusMessages = {
+    open: _('Opened during containment', 'covid19'),
+    open_as_usual: _('Opened during containment', 'covid19'),
+    maybe_open: _('Potentially opened during containment'),
```
Comment (bbecquet) [inline src/panel/poi/blocks/Covid19.jsx:110]: Not a big deal, just for consistency, but this one lacks the `'covid19'` context.

Comment (G-Ray) [reply src/panel/poi/blocks/Covid19.jsx:110]: definitely a miss :+1:

**Pull request merged**